### PR TITLE
KOGITO-4778: Removed Native image integration test from `process-user…

### PR DESCRIPTION
**Pull request by nmirasch**
…tasks-with-security-oidc-quarkus-with-console`

Jira: https://issues.redhat.com/browse/KOGITO-4778
Removed NativeApprovalsRestIT to fix native compilation on Process Integration Tests.

### CODE REVIEW

**Comment by cristianonicolai** [comment]
@nmirasch you also need to remove the module from the native profile https://github.com/kiegroup/kogito-examples/blob/master/pom.xml#L263

**Comment by radtriste** [comment]
Jenkins run native

File: pom.xml [hunk 1]
```diff
@@ -263,7 +263,6 @@
         <module>process-usertasks-quarkus-with-console</module>
         <module>process-usertasks-timer-quarkus-with-console</module>
         <module>process-usertasks-with-security-oidc-quarkus</module>
-        <module>process-usertasks-with-security-oidc-quarkus-with-console</module>
```
Comment (evacchi) [inline pom.xml:266]: are you sure you wanted to remove it from the main build?

Comment (nmirasch) [reply pom.xml:266]: @evacchi I removed it form native profile modules, the native image could still be generated inside the example but it's not part of the build.

**Comment by nmirasch** [comment]
Jenkins run native

**Comment by nmirasch** [comment]
Jenkins run native

**Comment by radtriste** [comment]
@nmirasch looks like the check does timeout ... can you please change the [timeout value](https://github.com/kiegroup/kogito-examples/blob/stable/Jenkinsfile#L19) to 540 minutes ?
native checks is quite demanding and takes time. and we have more and more examples ...

**Comment by MarianMacik** [comment]
Hi, this can be closed in favour of #632. I have fixed all issues there (3 in total).

**Comment by nmirasch** [comment]
Thanks @MarianMacik!

**Pull request closed without merging**